package-system/python/win_x64/build_python.bat

```diff
@@ -29,16 +29,21 @@ set vswhere_location=%ProgramFiles(x86)%\Microsoft Visual Studio\Installer
 echo adding %vswhere_location% to PATH
 set PATH=%vswhere_location%;%PATH%
 
-for /f "tokens=*" %%i in ('vswhere -version [15.0^,16.0^) -property installationPath') do set VS2017_LOCATION=%%i
+for /f "tokens=*" %%i in ('vswhere -version [16.0^,17.0^) -property installationPath') do set VS2017_LOCATION=%%i
```

Comment:
VS2017??

Reply:
The update is to use at least 2019. I just forgot to update the variable name